### PR TITLE
Release MSAL 1.6.0

### PR DESCRIPTION
**Pull request by iambmelt**
Staging PR for MSAL 1.6.0

## Release Candidate 1
- ~Update `changelog` for `1.6.0`~ 4556c702beea39396cd36b0b9a35a34248daad4a
- ~Update `version.properties` for `1.6.0-RC1`~ 5163edd2197da0885a284696872b210d4404ef5c
- ~Update `build.gradle` deps for `2.0.15-RC1`~ 9824680dff0ed30a3bdf755bcb3f151a3251ae91

## Release
- ~Update `version.properties` for `1.6.0`~ 796ce1a464ded8bf0d4ed14f7ed9fb76766c6875
- ~Update `build.gradle` deps for `2.0.15`~ 779ce60e58052a22cc53d20837eed93cad47ea46

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,5 +1,12 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 
+Version 1.6.0
+-----------
+- Introduces new AadAuthorityAudience enum to support new syntax for specifying cloud + audience
+- Adds new property to IAuthenticationResult: correlation_id
+- Fix for String encoding issue affecting devices with Turkish locale
+- Picks up common@2.0.15
```
Comment (shahzaibj) [inline changelog:8]: We also had a couple of bug fixes as well for following:

- https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/1092
- https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/1096

Comment (iambmelt) [reply changelog:8]: Thanks @shahzaibj -- will update

Comment (iambmelt) [reply changelog]: Updated in 914c0700c0fcf739b5b652e796091513199d3e16

File: msal/build.gradle [hunk 2]
```diff
@@ -148,9 +148,9 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '2.0.12-hf1', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '2.0.15-SNAPSHOT', changing: true)
```
Comment (kreedula) [inline msal/build.gradle:151]: needs to be `2.0.15`

**Pull request merged**